src/game/stage/collision.c

```diff
@@ -821,11 +821,12 @@ s32 sub_801EF94(s32 p0, s32 p1, s32 layer)
     coll = gRefCollision;
     mtIndex = ((u16 *)coll->map[layer])[(r3 * coll->levelX) + r8];
 
-#ifndef NON_MATCHING
     // ((r5 << 3) + (r5 << 2)) == r5 * TILES_PER_METATILE_AXIS
     r1 = ((r5 << 3) + (r5 << 2) + r7);
 
+#if !PORTABLE
```

Comment:
```suggestion
#ifndef NON_MATCHING
```

I think we discussed this has nothing to do with portability

Reply:
Yup!